src/pretix/static/pretixbase/js/addressform.js

```diff
@@ -5,6 +5,14 @@ $(function () {
     // to prevent fetching the same thing many times.
     var responseCache = {};
 
+    const cleanName = (name) => {
+        // Remove form prefix
+        name = name.split("-").pop();
+        // Remove settings prefix
+        name = name.replace(/^invoice_address_from_/, "");
```

Comment:
I could not test a case, where this was necessary. Do I need to install pretix_peppol?